statsmodels/tsa/x13.py

```diff
@@ -384,6 +384,10 @@ def x13_arima_analysis(endog, maxorder=(2, 1), maxdiff=(2, 1), diff=None,
         environmental variable. If False, will look for x12a first and will
         fallback to the X12PATH environmental variable. If x12path points
         to the path for the X12/X13 binary, it does nothing.
+    dir : str
+        The path to where temporary files are created by the function. 
+        If None, the function will create files in 
+        C:\Users\{usrname}\AppData\Local\Temp
```

Comment:
This isn't quite correct.  Should be 

```
If None, files are created in the default temporary file location.
```

This needs to be changed since it works on OSX and Linux, among other OSes.